spec/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/order.rb'
+
+describe Order do
+  it 'can i choose an item an store it?' do
+    subject.start_order(:hawaiian, 3)
+    expect(subject.current_order).to eq [{:hawaiian => 7, :item_no => 3}]
+  end
+  it 'can hold an order for more than one of a certain item' do
+    subject.start_order(:hawaiian, 2)
+    expect(subject.current_order).to eq [{:hawaiian => 7, :item_no => 2}]
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)